bus-mapping/src/evm/opcodes/callop.rs

```diff
@@ -173,9 +176,8 @@ impl<const N_ARGS: usize> Opcode for CallOpcode<N_ARGS> {
         let is_precompile = code_address
             .map(|ref addr| is_precompiled(addr))
             .unwrap_or(false);
-        // TODO: What about transfer for CALLCODE?
```

Comment:
I have this question about if CALLCODE should transfer, too.

The only answer I have now is the spec [says](https://github.com/privacy-scaling-explorations/zkevm-specs/blob/025cd17b97bfc32efd973eecd23c8eb289937264/specs/opcode/F1CALL_F2CALLCODE_F4DELEGATECALL_FASTATICCALL.md?plain=1#L81) only CALL transfers.

Reply:
i read the geth codes, i remember callcode will not transfer, but it will check balance for balance not enough err...

Reply:
What a strange behavior 😂. Hope we can remove CALLCODE in future forks.